src/Layout.tsx

```diff
@@ -3,9 +3,21 @@ import { Outlet, useSearchParams } from 'react-router-dom'
 import { nanoid } from 'nanoid'
 import { configureAbly } from '@ably-labs/react-hooks'
 import InfoCard from './InfoCard'
+import { viewRepoAtGithub, viewAllExamples, openNewWindow } from './gtm'
 
 const clientId = nanoid()
 
+import TagManager from 'react-gtm-module'
+
+const tagManagerArgs = {
+  gtmId: 'GTM-TZ37KKW',
```

Comment:
ideally we could pass this through an ENV variable, but that can be refactored too